src/include/daos_types.h

```diff
@@ -217,6 +218,30 @@ typedef struct {
 	uint64_t	hi;
 } daos_obj_id_t;
 
+#define DAOS_UUID_STR_SIZE 37	/* 36 + 1 for '\0' */
+
+static inline bool
+daos_is_valid_uuid_string(const char *uuid)
+{
+	const char	*p;
+	int		 len = DAOS_UUID_STR_SIZE - 1; /* Not include the ternimated '\0' */
+	int		 i;
+
+	if (strnlen(uuid, len) != len)
```

Comment:
[minor] more safe check might be if (strnlen(uuid, len + 1) != len), in case uuid's len is larger than len...

Reply:
Yes, this was original from daos_label_is_valid().